file/file.go

```diff
@@ -31,7 +32,9 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
 		switch msg.String() {
-		case "ctrl+c", "q":
+		case "ctrl+c", "q", "esc":
```

Comment:
I believe `esc` is used to go back when a user navigated into a directory.

```suggestion
		case "ctrl+c", "q":
```

Reply:
Isn't that a bit confusing when most gum programs cancel on escape? And you already can go back using the arrow keys.

Reply:
That's a fair point! I think that makes sense to me. I like the consistency and because there is already a way to go up / back. We can merge this as-is.